compose.yaml

```diff
@@ -5,3 +5,12 @@ services:
       - dev
     volumes:
       - ./:/md
+
+  jinja2-cli:
```

Comment:
What is this used for?

Reply:
It's used to generate the `README.workflows.md` files from `task/Taskfile.github-actions.yml`.